### DISCUSSION TITLE
postmarketos waydroid camera drivers

### DISCUSSION THREAD

**Discussion by datamaskinen** [waydroid/waydroid#1360] [General]
Hi,
My question is ..

If i have postmarketos installed on my phone as main os and my camera does not have any drivers for linux/postmarketos but i know that the camera works in android.

Can waydroid reach the camera hardware and use some android driver to use the camera in some way?

Passtrue the hardware in someway?

**Comment by Newbytee** [comment]
Even if you could pass through the camera hardware somehow, close-to-mainline kernels have different camera drivers than the vendor-provided Android Linux kernels, so it wouldn't work. In many cases there's also just no kernel driver at all.
